src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -127,3 +129,149 @@ def crop_roi_mask(roi_mask: coo_matrix) -> coo_matrix:
     cropped_mask = roi_mask.tocsr()[min_row:max_row, min_col:max_col]
 
     return cropped_mask.tocoo()
+
+
+def coo_rois_to_old(coo_masks: List[coo_matrix],
+                    max_correction_vals: motion_border,
+                    movie_shape: Tuple[int, int]) -> List[Dict[str, Any]]:
+    """
+    Converts coo formatted ROIs to old format from previous Ophys Segmentation
```

Comment:
Please be more descriptive than "old" and "previous". As soon as it gets changed again it's going to be unclear what this means.

Reply:
changed to 'lims_compatible'